inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/es/ElasticsearchResourceOperator.java

```diff
@@ -141,25 +139,7 @@ private ElasticsearchConfig getElasticsearchConfig(SinkInfo sinkInfo, Elasticsea
             config.setUsername(esInfo.getUsername());
             config.setPassword(esInfo.getPassword());
         }
-        config.setHost(esInfo.getHost());
-        config.setPort(esInfo.getPort());
-        // read from data node if not supplied by user
```

Comment:
Whether or not  to add the logic of reading from data node

Reply:
These configs used here comes from `stream_sink` table,   `stream_sink` table data comes from the data node and saved. I think may be it's ok to use directly here.